netci.groovy

```diff
@@ -528,7 +528,7 @@ branchList.each { branchName ->
             Utilities.addArchival(newBuildJob, "bin/build.pack,bin/osGroup.AnyCPU.${configurationGroup}/**,bin/ref/**,bin/packages/**,msbuild.log")
 
             // Use Server GC for Ubuntu/OSX Debug PR build & test
-            def useServerGC = ((os == 'Ubuntu' || os == 'OSX') && configurationGroup == 'Release' && isPR)
+            def useServerGC = ((os == 'Ubuntu15.10' || os == 'OSX') && configurationGroup == 'Release' && isPR)
```

Comment:
Yeah, that's true. Since we're not using server GC in the "regular build-test run", I figured I'd switch to using it in this run instead. The 14.04/15.10 runs should be very similar regardless.